src/utils.c

```diff
@@ -903,3 +903,30 @@ ts_subtract_integer_from_now(PG_FUNCTION_ARGS)
 	ts_cache_release(hcache);
 	return Int64GetDatum(res);
 }
+
+RelationSize
+ts_relation_size(Oid relid)
+{
+	int64 tot_size;
+	int i = 0;
+	RelationSize ret;
+	Datum oid = ObjectIdGetDatum(relid);
+	static const char *filtyp[] = { "main", "init", "fsm", "vm" };
+
+	/* for heap get size from fsm, vm, init and main as this is included in
```

Comment:
Nit picking: Most `multi-line` comments are starting on the same line as `\*` whereas they should start on the next line.

Reply:
I just moved this code from another file, but will update the comment.

Reply:
We should try to update the clang formatter to ensure multi-line comments are formatted correctly.